normative-rules.md

```diff
@@ -0,0 +1,94 @@
+# How to Tag Normative Rules in RISC-V International Standards
+
+## What is a Normative Rule?
+
+Normative rules specify the behaviors an implementation must meet in order to be compliant with the standard. Each normative rule can be thought of as a complete individual testable behavior. In some cases, normative rules allow a plurality of acceptable implementations. These are called "parameters" and can be thought of as a special case of a normative rule.
+
+### Examples of Normative Rules:
+| ISA Manual Text |
+| --------------- |
+| "For RV32I, the 32 `x` registers are each 32 bits wide, i.e., `XLEN=32`." |
+| "Register `x0` is hardwired with all bits equal to 0." |
+| "ADDI adds the sign-extended 12-bit immediate to register _rs1_." |
+| "M-mode code can access all CSRs at lower privilege levels." |
+
+### Examples of Parameter Normative Rules:
+
+| ISA Manual Text | Type | Allowed Values |
+| --------------- | ---- | -------------- |
+| "The EEI will define whether the memory system is little-endian or big-endian." | Enum | little or big |
+| "The `misa` register must be readable in any implementation, but a value of zero can be returned to indicate the `misa` register has not been implemented." | Boolean | true/false |
+| "If `misa` is nonzero, the MXL field indicates the effective XLEN in M-mode, a constant termed _MXLEN_." | Integer | 32 or 64 |
```

Comment:
Interesting problem: the MXL field is 2 bits, and can't hold 32 or 64. It is really an enum with encodings that map to 32 and 64.